source/_integrations/template.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: Template
+title: Template Sensor
```

Comment:
This title is automatically set from the manifest files of integrations. Do you have a PR on the codebase that changes this?

If not, changing this, will not work.